addons/binding/org.openhab.binding.oceanic/OSGI-INF/OceanicHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
Karel-Goderis-MacBook-Air:openhab2-addons kgoderis$ mvn license:format
[INFO] Scanning for projects...
[WARNING] Error injecting: org.eclipse.tycho.core.p2.P2ArtifactRepositoryLayout
java.lang.TypeNotPresentException: Type org.eclipse.tycho.core.p2.P2ArtifactRepositoryLayout not present